js-api-spec.ts

```diff
@@ -54,26 +59,18 @@ del.sync(dir, {force: true});
 const sassPackagePath = p.join(dir, 'node_modules', 'sass');
 fs.mkdirSync(sassPackagePath, {recursive: true});
 
-const configPath = p.join(dir, 'jest.config.json');
-fs.writeFileSync(
-  configPath,
-  JSON.stringify({
-    ...config,
-    rootDir: p.resolve('.'),
-    roots: ['<rootDir>/js-api-spec'],
-  })
-);
-
 // We want to use the type information from --sassSassRepo even if --sassPackage
 // is written in TypeScript, because the specs may test behavior that's not yet
 // implemented in --sassPackage and we don't want that to cause a compile error.
 // We accomplish this by creating a JavaScript package named "sass" that
 // requires and re-exports --sassPackage, and using the type annotations from
 // --sassSassRepo as that package's annotations.
-const packageRequire = JSON.stringify(p.resolve(argv.sassPackage));
+const packageRequire = argv.browser
+  ? p.resolve(argv.sassPackage, 'sass.default.js')
```

Comment:
In the browser context, we want to directly import the compiled browser version.